### PR TITLE
updated nuget packages

### PR DESCRIPTION
**Pull request by VladislavMorozovIDT**

### CODE REVIEW

File: src/SqsPoller.Extensions.Publisher/SqsPoller.Extensions.Publisher.csproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <PropertyGroup>
         <TargetFramework>netstandard2.0</TargetFramework>
         <PackageId>SqsPoller.Extensions.Publisher</PackageId>
-        <Version>0.13.0$(VersionSuffix)</Version>
+        <Version>0.13.1$(VersionSuffix)</Version>
```
Comment (dsharkou) [inline src/SqsPoller.Extensions.Publisher/SqsPoller.Extensions.Publisher.csproj]: Please specify `0.14.0`

File: src/SqsPoller/SqsPoller.csproj [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <PropertyGroup>
         <TargetFramework>netstandard2.0</TargetFramework>
         <PackageId>SqsPoller</PackageId>
-        <Version>0.13.0$(VersionSuffix)</Version>
+        <Version>0.13.1$(VersionSuffix)</Version>
```
Comment (dsharkou) [inline src/SqsPoller/SqsPoller.csproj]: Please specify `0.14.0`

File: src/SqsPoller/SqsPoller.csproj [hunk 3]
```diff
@@ -25,8 +25,8 @@
     
     <ItemGroup>
         <PackageReference Include="AWSSDK.SQS" Version="3.7.0.14" />
-        <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="5.0.0" />
-        <PackageReference Include="Microsoft.Extensions.Hosting" Version="5.0.0" />
+        <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
+        <PackageReference Include="Microsoft.Extensions.Hosting" Version="6.0.0" />
         <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```
Comment (dsharkou) [inline src/SqsPoller/SqsPoller.csproj:29]: Ideally to get rid of `Newtonsoft.Json` and use `System.Text.Json.JsonSerializer`. But it's optional

**Review by dsharkou — COMMENTED** [review]
Please fix comments

File: src/SqsPoller/SqsPoller.csproj (hunk 3 above)
Comment (VladislavMorozovIDT) [reply src/SqsPoller/SqsPoller.csproj:29]: It needs to refactor the logic of library

File: src/SqsPoller/SqsPoller.csproj (hunk 2 above)
Comment (VladislavMorozovIDT) [reply src/SqsPoller/SqsPoller.csproj]: fixed

File: src/SqsPoller.Extensions.Publisher/SqsPoller.Extensions.Publisher.csproj (hunk 1 above)
Comment (VladislavMorozovIDT) [reply src/SqsPoller.Extensions.Publisher/SqsPoller.Extensions.Publisher.csproj]: fixed

File: src/SqsPoller/SqsPoller.csproj (hunk 3 above)
Comment (dsharkou) [reply src/SqsPoller/SqsPoller.csproj:29]: next PR

**Review by dsharkou — APPROVED** [review]

**Review by dsharkou — CHANGES_REQUESTED** [review]
Please update to `.net5` to `.net6`

**Review by dsharkou — APPROVED** [review]

**Pull request merged**